settings.gradle

```diff
@@ -1 +1,44 @@
 include ':app'
+
+Properties localProperties = null;
+String settingAppServicesPath = "substitutions.appServicesPath";
```

Comment:
Can we make this `substitutions.application-services.dir`?  We have `sdk.dir` and `ndk.dir`, let's follow suit.  And if we can do this, then we can make grep work that little bit better.

Ditto below.